File: Compilers-2/Mini-Asgn-1_ES18BTECH11028/exceptionhandling.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int a=2;
    int b=01;
    try
    {
        if(b==0) throw 0;
        cout<<a/b<<endl;
    }
    catch(int x)
    {
        cout<<"Denominator is zero. Please provide another number as denominator"<<endl;
    }
    /*As we can see if b=0 then immediatly it passes the comand to catch 
    block and the chunck of code present there is executed */

    return 0;
}

File: Compilers-2/Mini-Asgn-1_ES18BTECH11028/exceptionhandling.py
#Code to show exception handling in Python
a=2
b=0

try:
    print(a/b)
except Exception as e:
    print("Denominator is zero. Please provide another number as denominator")
#As we can see when we pass 0 as denominartor it goes inside 
#the except block and executes the code there.